server/middleware/users.test.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import { NextFunction } from 'express-async-router';
+
+import { removeMarkup } from './users';
+
+
+describe('removeMarkup middleware', () => {
+  it('should remove markup from request body fields', () => {
```

Comment:
Should we actually be removing the markup or converting it to HTML entities instead?